resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -7,4 +7,4 @@ if [ ! -e "$JRUBY_JAR" ]; then
   return 1
 fi
 
-CLASSPATH="${CLASSPATH}:${JRUBY_JAR}"
+CLASSPATH="${CLASSPATH}:${JRUBY_JAR}:/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/jars/*"
```

Comment:
Yeah, we might want to change the location. In general we (in the past, in puppetserver) put things under /opt/ that we don't intend the user to have to modify. Things we expect the user to modify have gone under /etc/puppetlabs/, and that might be better just because the user will have all their configuration there already. On the other hand we put all the other binaries under /opt/, so ¯\\\_(ツ)\_/¯